pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -0,0 +1,48 @@
+{
+  stdenv, lib, fetchFromGitHub, makeDesktopItem, prusa-slicer
+}:
+let
+  appname = "SuperSlicer";
+  version = "2.2.53.1";
+  pname = "super-slicer";
+  override = super: {
+    inherit version pname;
+
+    src = fetchFromGitHub {
+      owner = "supermerill";
+      repo = "SuperSlicer";
+      hash = "sha256-CAhwmQ63N/XJYToTnIV84lNnjDGNbkmYPzNKNL/wVxs=";
+      rev = "${version}";
```

Comment:
`rev = version;` is more idiomatic

Reply:
oof, yeah, that came from line formatting before)